templates/cloud/index.html

```diff
@@ -64,15 +64,15 @@
     </div>
     <div class="twelve-col vertical-divider">
         <div class="four-col">
```

Comment:
Should you be using Vanilla grid here?

Reply:
I'll address this when I get to the /cloud section